dataeng/resources/remote-config.sh

```diff
@@ -91,11 +91,29 @@ unassume_role
 #         usernamePassword('ANALYTICS_VAULT_ROLE_ID', 'ANALYTICS_VAULT_SECRET_ID', 'analytics-vault');
 #     }
 # }
+# set path for config file which store token in each job workspace
+export VAULT_CONFIG_PATH= ${WORKSPACE}/vault-config/vault_config
```

Comment:
Just a heads-up that the space after the equal sign is not valid bash or sh syntax.